src/components/Indicator/Indicator.tsx

```diff
@@ -2,14 +2,20 @@ import * as React from 'react';
 import {classNames} from '@shopify/css-utilities';
 import styles from './Indicator.scss';
 
+type Tone = 'positive' | 'neutral' | 'negative';
+
 export interface Props {
+  // Defaults to positive
+  tone?: Tone;
+  // Sets an animated transparent background
   pulse?: boolean;
 }
 
-export default function Indicator({pulse = true}: Props) {
```

Comment:
Was there a requirement for removing the default of true? Just thinking default of true makes sense based on the nature of the component and technically this could be an unexpected change for people using it. What do you think?

Reply:
Was wondering about this as well but it caught me off guard when trying to use it. I'm fine with defaulting to true if you think that's best. It's less breaking than defaulting to false